.github/workflows/pythonpackage.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       run: |
         git clone https://github.com/nevermined-io/rapidsnark
         cd rapidsnark
+        git checkout v0.0.1
```

Comment:
maybe this avoids cloning the whole `rapidsnark` repo if you only need `v0.0.1` branch:
```
git clone --branch v0.0.1 https://github.com/nevermined-io/rapidsnark
```